<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pg_gestao_estoque</class>
 <widget class="QDialog" name="pg_gestao_estoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>891</width>
     <height>681</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="novo_produto">
    <attribute name="title">
     <string>Novo Produto</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>791</width>
       <height>534</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="cod_produto">
          <property name="text">
           <string>Código do Produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="campo_codigo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="cod_produto_2">
          <property name="text">
           <string>Descrição do Produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="campo_descricao"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="cod_produto_3">
            <property name="text">
             <string>Valor de Venda</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="campo_valor_venda"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="cod_produto_4">
            <property name="text">
             <string>Valor de Compra</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="campo_valor_compra"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="cod_produto_5">
            <property name="text">
             <string>Quantidade em Estoque</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="campo_quantidade"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="cod_produto_6">
            <property name="text">
             <string>Fornecedor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="campo_fornecedor"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>300</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>420</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_novo_produto">
          <property name="text">
           <string>Novo </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_salvar">
          <property name="text">
           <string>Salvar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="gestao_produtos">
    <attribute name="title">
     <string>Gestão Produtos</string>
    </attribute>
    <widget class="QTableWidget" name="tw_produtos">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>150</y>
       <width>381</width>
       <height>481</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>171</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Filtrar por:</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>175</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QRadioButton" name="filtro_cod_produto">
         <property name="text">
          <string>Código do Produto</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="filtro_des_produto">
         <property name="text">
          <string>Descrição do Produto</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>110</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="campo_pesquisar">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_pesquisar">
        <property name="text">
         <string>Pesquisar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>481</width>
       <height>624</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_15">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="cod_produto_7">
          <property name="text">
           <string>Código do Produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="campo_codigo2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLabel" name="produto">
          <property name="text">
           <string>Produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="campo_produto2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QLabel" name="quantidade">
          <property name="text">
           <string>Quantidade</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="campo_quantidade2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QLabel" name="valor_compra">
          <property name="text">
           <string>Valor Compra</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="campo_valor_compra2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QLabel" name="valor_venda">
          <property name="text">
           <string>Valor Venda</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="campo_valor_venda2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <widget class="QLabel" name="fornecedor">
          <property name="text">
           <string>Fornecedor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="campo_fornecedor2"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="btn_salvar2">
          <property name="text">
           <string>Salvar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_excluir">
          <property name="text">
           <string>Excluir</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
